src/esp/bindings/SensorBindings.cpp

```diff
@@ -93,11 +98,29 @@ void initSensorBindings(py::module& m) {
       .def_readwrite("clear_color", &CameraSensorSpec::clearColor);
 
   // ====CameraSensorSpec ====
-  py::class_<CameraSensorSpec, CameraSensorSpec::ptr, VisualSensorSpec,
-             SensorSpec>(m, "CameraSensorSpec", py::dynamic_attr())
+  py::class_<CameraSensorSpec, CameraSensorSpec::ptr, VisualSensorSpec>(
+      m, "CameraSensorSpec", py::dynamic_attr())
       .def(py::init(&CameraSensorSpec::create<>))
       .def_readwrite("ortho_scale", &CameraSensorSpec::orthoScale);
 
+  // ====FisheyeSensorSpec ====
+  py::class_<FisheyeSensorSpec, FisheyeSensorSpec::ptr, VisualSensorSpec>(
+      m, "FisheyeSensorSpec", py::dynamic_attr())
+      .def(py::init(&FisheyeSensorSpec::create<>))
+      .def_readwrite("focal_length", &FisheyeSensorSpec::focalLength)
+      .def_readwrite("principal_point_offset",
+                     &FisheyeSensorSpec::principalPointOffset)
+      .def_readwrite("cubemap_size", &FisheyeSensorSpec::cubemapSize)
+      .def_readwrite("sensor_subtype", &FisheyeSensorSpec::fisheyeModelType);
+
+  // ====FisheyeSensorDoubleSphereSpec ====
```

Comment:
You can consider leaving a comment on the parameters and pointing to the paper:
  /**
   * alpha and xi are specific to "double sphere" camera model.
   * see details (value ranges) in:
   * Vladyslav Usenko, Nikolaus Demmel and Daniel Cremers: The Double Sphere
   * Camera Model, The International Conference on 3D Vision (3DV), 2018
   */